vms/volunteer/templates/volunteer/search.html

```diff
@@ -137,7 +137,12 @@
                     <div class="form-group">
                         <label class="col-md-2 control-label">{% trans "Organization" %}</label>
                         <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Search by Organization{% endblocktrans %}" type="text" name="organization" value="{% if form.organization.value %}{{ form.organization.value }}{% endif %}">
+                            <select id="select" class="form-control" name="organization">
+								<option value="">-- {% trans "Select Organization" %} --</option>
```

Comment:
👋 Did you test this for any other language than English, say French?

Reply:
@tapasweni-pathak No. I didn't test for any other language. Please suggest how i proceed.

Reply:
`trans` works here, just checked if we stick to `blocktrans` or not. Perfect!